src/Components/ShareCipher/shareCipher.scss

```
@@ -14,3 +14,15 @@
 .create-link * {
     transition: all .3s ease-out;
 }
+
+.center{
+    justify-content: center;
+    align-content: center;
+    align-items: center;
+    text-align: center;
+}
+
+.cipher-container {
+    min-width: 60vw;
+    max-width: 60vw;
```

```suggestion
    width: 60vw;
```

You can use `width: 60vw` here :+1: